packages/compass-components/src/components/options-dropdown-button.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useMemo } from 'react';
+import { Link, Icon } from './leafygreen';
+import { css } from '@leafygreen-ui/emotion';
+import { spacing } from '@leafygreen-ui/tokens';
+import { uiColors } from '@leafygreen-ui/palette';
+
+const optionsButtonStyles = css({
+  backgroundColor: 'transparent',
+  border: 'none',
+  display: 'inline',
+  height: spacing[4] + spacing[1],
+  ':focus': {
+    outline: `${spacing[1]}px auto ${uiColors.focus}`,
+  },
+});
+
+const optionStyles = css({
+  display: 'flex',
+  alignItems: 'center',
+});
+
+type OptionsDropdownButtonProps = {
+  'aria-controls': string;
+  'data-testid'?: string;
+  isExpanded: boolean;
+  onToggleOptions: () => void;
+  id?: string;
+};
+
+export const OptionsDropdownButton: React.FunctionComponent<OptionsDropdownButtonProps> =
+  ({
+    'aria-controls': ariaControls,
+    isExpanded,
+    id,
+    'data-testid': dataTestId,
+    onToggleOptions,
+  }) => {
+    const optionsIcon = useMemo(
+      () => (isExpanded ? 'CaretDown' : 'CaretRight'),
+      [isExpanded]
+    );
+    const optionsLabel = useMemo(
+      () => (isExpanded ? 'Less Options' : 'More Options'),
+      [isExpanded]
+    );
+    return (
+      <Link
+        aria-label={optionsLabel}
+        aria-expanded={isExpanded}
+        aria-controls={ariaControls}
+        id={id}
+        as="button"
+        className={optionsButtonStyles}
+        data-testid={dataTestId}
+        hideExternalIcon={true}
+        onClick={onToggleOptions}
+      >
+        <div className={optionStyles}>
+          {optionsLabel}
```

Comment:
If the label gets too big, do we want to handle text overflow or are we okay with this component getting too long or allowing text to break on a new line? If this is a generic component, we probably want to handle it in a way that would work for different use cases and document it

Reply:
Ignore this, label is always just two options that are part of the component implementation so obviously it can't get that long, my bad